<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeVacation</class>
 <widget class="QWidget" name="EmployeeVacation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>姓名:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_name">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>批准人:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_auditor">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>请假日期:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit_leaveStart">
             <property name="date">
              <date>
               <year>2014</year>
               <month>3</month>
               <day>1</day>
              </date>
             </property>
             <property name="maximumDate">
              <date>
               <year>2050</year>
               <month>12</month>
               <day>31</day>
              </date>
             </property>
             <property name="minimumDate">
              <date>
               <year>2014</year>
               <month>3</month>
               <day>1</day>
              </date>
             </property>
             <property name="displayFormat">
              <string>yyyy-MM-dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit_leaveEnd">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2050</year>
               <month>12</month>
               <day>31</day>
              </datetime>
             </property>
             <property name="date">
              <date>
               <year>2050</year>
               <month>12</month>
               <day>31</day>
              </date>
             </property>
             <property name="maximumDate">
              <date>
               <year>2050</year>
               <month>12</month>
               <day>31</day>
              </date>
             </property>
             <property name="minimumDate">
              <date>
               <year>2014</year>
               <month>3</month>
               <day>1</day>
              </date>
             </property>
             <property name="displayFormat">
              <string>yyyy-MM-dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>销假日期:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit_backStart">
             <property name="date">
              <date>
               <year>2014</year>
               <month>3</month>
               <day>1</day>
              </date>
             </property>
             <property name="maximumDate">
              <date>
               <year>2050</year>
               <month>12</month>
               <day>31</day>
              </date>
             </property>
             <property name="minimumDate">
              <date>
               <year>2014</year>
               <month>3</month>
               <day>1</day>
              </date>
             </property>
             <property name="displayFormat">
              <string>yyyy-MM-dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit_backEnd">
             <property name="date">
              <date>
               <year>2050</year>
               <month>12</month>
               <day>31</day>
              </date>
             </property>
             <property name="maximumDate">
              <date>
               <year>2050</year>
               <month>12</month>
               <day>31</day>
              </date>
             </property>
             <property name="minimumDate">
              <date>
               <year>2014</year>
               <month>3</month>
               <day>1</day>
              </date>
             </property>
             <property name="displayFormat">
              <string>yyyy-MM-dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>操作日期:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit_start">
             <property name="date">
              <date>
               <year>2014</year>
               <month>3</month>
               <day>1</day>
              </date>
             </property>
             <property name="maximumDate">
              <date>
               <year>2050</year>
               <month>12</month>
               <day>31</day>
              </date>
             </property>
             <property name="minimumDate">
              <date>
               <year>2014</year>
               <month>3</month>
               <day>1</day>
              </date>
             </property>
             <property name="displayFormat">
              <string>yyyy-MM-dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit_end">
             <property name="date">
              <date>
               <year>2050</year>
               <month>12</month>
               <day>31</day>
              </date>
             </property>
             <property name="maximumDate">
              <date>
               <year>2050</year>
               <month>12</month>
               <day>31</day>
              </date>
             </property>
             <property name="minimumDate">
              <date>
               <year>2014</year>
               <month>3</month>
               <day>1</day>
              </date>
             </property>
             <property name="displayFormat">
              <string>yyyy-MM-dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>状态:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_status">
               <item>
                <property name="text">
                 <string>全部</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>已请假</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>已销假</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btn_clear">
             <property name="text">
              <string>清筛选条件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_search">
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_add">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_change">
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_delete">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_close">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
